src/wp-includes/rest-api/endpoints/class-wp-rest-users-controller.php

```diff
@@ -1131,7 +1147,7 @@ public function check_user_password( $value, $request, $param ) {
 			return new WP_Error( 'rest_user_invalid_password', __( 'Passwords cannot be empty.' ), array( 'status' => 400 ) );
 		}
 
-		if ( false !== strpos( $password, "\\" ) ) {
+		if ( false !== strpos( $password, '\\' ) ) {
```

Comment:
Single quote is not equivalent to double quote for the slashes.

Reply:
I think it may depend on the individual use scenario, single quotes are used upstream:
https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/endpoints/class-wp-rest-users-controller.php#L1317

Reply:
But the logic is not correct, so it could be an error upstream.

Reply:
I was misreading it. My mistake.